babel/src/board.js

```diff
@@ -35,6 +31,10 @@ export class Board {
         return triples.combine(this.triples);
     }
     
+    get atticOrbs() {
+        return _.cloneDeep(new Board(this.width, this.height, this.atticTypes).orbs);
+    }
```

Comment:
Maybe this would work?

```js
b = new Board();
c = b.attic;
c.types = 'abcdefgh'.split('');
c.generateOrbs();
```

The last line might need to be `b.attic = c;`, but I think this is unnecessary because the `attic` object would be a reference, not a value. By updating the attic object `c`, you are also modifying the the same object `b.attic`.